<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>wbsell</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Resources/images/logo.png</normaloff>../Resources/images/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>1141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="logoLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Resources/images/logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
      </font>
     </property>
     <property name="text">
      <string>wbsell</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trackingButton">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Resources/images/start.png</normaloff>../Resources/images/start.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="settingsButton">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Resources/images/settings.png</normaloff>../Resources/images/settings.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="notificationsButton">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Resources/images/notifications.png</normaloff>../Resources/images/notifications.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="settingsFrame">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>140</y>
      <width>281</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="settingsLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Настройки</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="key">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>400</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="keyLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>API-ключ</string>
     </property>
    </widget>
    <widget class="QLabel" name="themeLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Тема</string>
     </property>
    </widget>
    <widget class="QComboBox" name="theme">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Светлая</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Темная</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="dateLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Первый вход</string>
     </property>
    </widget>
    <widget class="QLabel" name="date">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>MM/DD/YYYY</string>
     </property>
    </widget>
    <widget class="QLabel" name="logsLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Запись логов</string>
     </property>
    </widget>
    <widget class="QComboBox" name="logsBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Да</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Нет</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="collapseButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Resources/images/collapse.png</normaloff>../Resources/images/collapse.png</iconset>
     </property>
    </widget>
    <widget class="QFrame" name="collapseFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>settingsLabel</zorder>
    <zorder>keyLabel</zorder>
    <zorder>themeLabel</zorder>
    <zorder>dateLabel</zorder>
    <zorder>date</zorder>
    <zorder>theme</zorder>
    <zorder>key</zorder>
    <zorder>logsLabel</zorder>
    <zorder>logsBox</zorder>
    <zorder>collapseButton</zorder>
    <zorder>collapseFrame</zorder>
   </widget>
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>461</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Статистика</string>
     </attribute>
     <widget class="QPushButton" name="reloadstatsButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../Resources/images/reload.png</normaloff>../Resources/images/reload.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QTextEdit" name="stats">
      <property name="geometry">
       <rect>
        <x>-7</x>
        <y>-3</y>
        <width>471</width>
        <height>341</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>stats</zorder>
     <zorder>reloadstatsButton</zorder>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Логи</string>
     </attribute>
     <widget class="QPushButton" name="reloadlogsButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../Resources/images/reload.png</normaloff>../Resources/images/reload.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QTextEdit" name="logs">
      <property name="geometry">
       <rect>
        <x>-7</x>
        <y>-3</y>
        <width>471</width>
        <height>341</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>logs</zorder>
     <zorder>reloadlogsButton</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="connectionLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>550</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отсутствует подключение к интернету!</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="wrongkeyLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>550</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Некорректный API-ключ!</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="Wiki"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="report_a_bug"/>
    <addaction name="feedback"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionReport_a_bug">
   <property name="text">
    <string>Report a bug</string>
   </property>
  </action>
  <action name="actionOpen_README">
   <property name="text">
    <string>Open README</string>
   </property>
  </action>
  <action name="Wiki">
   <property name="text">
    <string>README проекта</string>
   </property>
  </action>
  <action name="report_a_bug">
   <property name="text">
    <string>Сообщить об ошибке</string>
   </property>
  </action>
  <action name="feedback">
   <property name="text">
    <string>Связь с разработчиком</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
